### DISCUSSION TITLE
Can I use Postgresql or MongoDB with embedjs

### DISCUSSION THREAD

**Discussion by nishantshah977** [llm-tools/embedJs#7] [General]
hello there, if yes can you please provide me with reference and best practices for using it. Also, I would like to know about it's architecture

**Comment by adhityan** [comment]
Hello Nishant, at this point there is no support for Postgres vector extensions (pgvector) or MongoDB. This will be eventually added but right now the focus is adding support for Cosmos DB and other Azure services.

If you really want to add support for these database, feel free to send in a PR. You can read more about adding support for custom vector databases in the documentation. If you send in a PR, I am appy to review it and add documentation.

**Comment by adhityan** [comment]
MongoDB (Atlas) is now supported by embedJs.

**Reply by nishantshah977** [reply]
That's great. I'm patiently waiting for Postgresql (Pgvector)
